### ISSUE TITLE
Are you planning to publish it ?

### ISSUE THREAD

**Issue by VessoVit** [liuhantang/DeepFacade#1]

**Comment by liuhantang** [comment]
Yes, I'll push the code later. Currently, the journal version of the paper is under review, and I need to further clean the code for publication.

**Comment by ThJOD** [comment]
Do you have an estimated time frame for the publishing of the code?

**Comment by wohaiyo** [comment]
Hello, when do you publish your code？

**Comment by yiqing427** [comment]
When are you planing to publish the code?
